WebRoot/js/zimbraAdmin/servers/model/ZaServer.js

```diff
@@ -878,6 +879,15 @@ ZaServer.modifyMethod = function (tmpObj) {
 			}
 			
 		}
+	}
+	
+	if(ZaItem.modifyMethodsExt["ZaServer"]) {
+		var methods = ZaItem.modifyMethodsExt["ZaServer"];
+		var cnt = methods.length;
+		for(var i = 0; i < cnt; i++) {
+			if(typeof(methods[i]) == "function")
+				methods[i].call(this, tmpObj, soapDoc);
```

Comment:
from where we are getting this tmpObj and soapDoc? is this function defined inside another function which passes this information?